crypto/curve25519/asm/x25519-asm-x86_64.S

```diff
@@ -27,6 +27,10 @@
 /* OS X's C ABI prefixes functions with underscore. */
 #define C_ABI(x) _ ## x
 #define HIDDEN .private_extern
+#elif defined(__MINGW32__) || defined(__MINGW64__)
+/* MinGW build support */
+#define C_ABI(x) x
+#define HIDDEN .globl
```

Comment:
I think this should be just `define HIDDEN`, based on what perlasm does. I'll make this change. Please test it and let me know if it breaks things.